### ISSUE TITLE
DELETE EDGE not removing all identical lightweightEdges

### ISSUE THREAD

**Issue by sinfex** [orientechnologies/orientdb#5605]
Two vertices, two identical edges between them, DELETE EDGE removes only one at a time.
Initially I suspected LIMIT defaults to 1, but apparently that is not the case.
This is only the case when _useLightweightEdges_ is enabled

```
orientdb {db=test}> ALTER DATABASE CUSTOM useLightweightEdges=true

Database updated successfully

orientdb {db=test}> CREATE VERTEX V ; CREATE VERTEX V

Created vertex 'V#9:0 v1'
Created vertex 'V#9:1 v1'

orientdb {db=test}> CREATE EDGE E FROM #9:0 TO #9:1 ; CREATE EDGE E FROM #9:0 TO #9:1

Created edge '[E{in:#9:1,out:#9:0}]'
Created edge '[E{in:#9:1,out:#9:0}]'

orientdb {db=test}> DELETE EDGE FROM #9:0 TO #9:1 LIMIT 2

Delete record(s) '1'

orientdb {db=test}> DELETE EDGE FROM #9:0 TO #9:1

Delete record(s) '1'

```

**Comment by luigidellaquila** [comment]
Hi @sinfex 

which OrientDB version?

Thanks

Luigi

**Comment by sinfex** [comment]
@luigidellaquila my apologies, OrientDB version: 2.1.8

**Comment by luigidellaquila** [comment]
got it, thanks. I'll test it asap

Luigi

**Comment by luigidellaquila** [comment]
Hi @sinfex 

I fixed it in 2.1.x branch, the fix will be released with 2.1.10

Thanks

Luigi
